capstone/src/main/java/com/google/step/data/RestaurantHeader.java

```diff
@@ -0,0 +1,99 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.step.data;
+
+import com.google.appengine.api.datastore.GeoPt;
+import org.apache.commons.lang3.builder.HashCodeBuilder;
+
+import java.util.List;
+
+
+/**
+ * A {@code RestaurantHeader} is an object that hold as subset of information about a restaurant.
+ * It holds a {@code Restaurant.restaurantKey}, a {@code Restaurant.name} a {@code Restaurant.location},
+ * and a {@code Restaurant.cuisine}.
+ */
+public final class RestaurantHeader {
+  private final long restaurantKey;
+  private final String name;
+  private final GeoPt location;
+  private final List<String> cuisine;
+
+  private RestaurantHeader(long restaurantKey,
+                           String name,
+                           GeoPt location,
+                           List<String> cuisine) {
+    this.restaurantKey = restaurantKey;
+    this.name = name;
+    this.location = location;
+    this.cuisine = cuisine;
```

Comment:
Based on these fields, I believe the documentation of the RestaurantsServlet doGet method needs to be updated. Would you mind updating it in this pull request?

Reply:
Can I update it in a separate pull request, as this pull request is simply for the creation of the restaurant header. Perhaps I could update it in the pull request that connects search to the servlet?

Reply:
Sure, that sounds like a good time to update.